### ISSUE TITLE
Parameter Issue

### ISSUE THREAD

**Issue by Pengxiao-Wang** [janesjanes/Pytorch-TextureGAN#2]
Hi,
Thank you for this research — I was confused with the parameter differences between the original paper and what you ilustrated in README (e.g. the weight of each loss, lr ...).  Above all, was the parameter "local_texture_size" supposed to be '-1' in the first training state, which you took '50' in both states?

**Comment by HideUnderBush** [comment]
Hi,
I got the same problem, and still failed to train a useful model which can provide the similar output as that given by the official model. Do you guys have any suggestions or hints on parameters and required epochs?

Thanks,

**Comment by chandrakant-sonawane** [comment]
@Pengxiao-Wang @HideUnderBush 
Have you got the solution for "style_weight" and other parameters, what should be their values?

**Comment by kmaeii** [comment]
Hi,
I got the same problem, could you share your parameters for me at pre-train and fine-tuning? @Pengxiao-Wang @HideUnderBush @chandrakant-sonawane
